src/content/cloud/container-resources.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Container Resources
+description: Optimizing Container Resource Allocation for Efficient Development Environments
+sidebar_label: Container Resources
+id: container-resources
+---
+
+import Image from '@theme/Image';
+
+Okteto leverages the concept of Kubernetes resources to decide on which node to place each container.
+When you specify a resource `limit` for a container, Okteto enforces those limits so that the running container is not allowed to use more of that resource than the limit you set.
+Okteto also reserves at least the `request` amount of that system resources specifically for that container to use.
+Follow [this link](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) to know more about how Kubernetes resources work.
+
+## Best Practices
+
+Okteto monitors the CPU and Memory usage of your development environments to balance containers across all the nodes in the cluster.
+However, it's not capable of evicting containers when a node becomes overloaded.
+To avoid this situation, it's strongly recommended that you configure your CPU and Memory `requests` to align with your container's actual resource utilization.
+By doing so, you ensure that your containers always have sufficient resources available for optimal development speed.
+
+## Resource Consumption Warnings
+
+The Okteto UI shows warnings to assist you in setting the right resources for your development environments.
+If a development environment is utilizing over 50% of its CPU or Memory requests, you'll receive a yellow warning:
```

Comment:
```suggestion
If a development environment is utilizing over 50% of its CPU or Memory requests, the Okteto UI will show a yellow warning:
```

Modifying this copy to make it about what the UI shows vs the customer "receiving" a warning. Small change that sounds softer to me 😄